<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>621</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Образовательный стандарт</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Компетенции</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Дисциплины</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_7"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Дисциплины</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Дисциплины учебного плана</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Компетенции</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Компетенции дисциплины</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Образовательный стандарт</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Образовательная программа</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Результат</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
